test/test_expansion.bats

```diff
@@ -89,3 +89,15 @@ function setup {
   assert_jq_contains '.jobs["cloudfoundry/blue_green"].steps[3].run.command' 'cf set-env blueskygreenbuilds-dark CIRCLE_BUILD_NUM ${CIRCLE_BUILD_NUM}'
 
 }
+
+@test "Job: blank subdomains are not in commands" {
+  # given
+  process_config_with test/inputs/jobs-separate_no-subdomain.yml
+
+  # when
+  assert_jq_contains '.jobs["cloudfoundry/dark_deploy"].steps[3].run.command' 'cf push --no-start "blueskygreenbuilds-dark" -f "/tmp/cf-manifest.yml" -p "/tmp/standalone-app.jar" -d "dark-blueskygreenbuilds.com"'
+  assert_jq_contains '.jobs["cloudfoundry/dark_deploy"].steps[3].run.command' 'cf unmap-route "blueskygreenbuilds" "dark-blueskygreenbuilds.com"'
+
+  assert_jq_contains '.jobs["cloudfoundry/live_deploy"].steps[1].run.command' 'cf map-route "blueskygreenbuilds-dark" "blueskygreenbuilds.com"'
```

Comment:
Note that this and the subsequent assert will unfortunately pass even before this change since they don't assert the absence of the `-n <<parameters.dark_subdomain>>` part of the line. It would probably be better overall if tests could assert that a whole line (shell command) is exactly something and nothing more.

Reply:
@shakuzen - you can use `assert_jq_match` to match the entire run command, but that would be several lines (unless we broke them into separate steps).

We could also add a `assert_jq_not_contains` to ensure text is not present.

I'm not concerned about which path, but I agree we need to tighten up validation here

Reply:
I attempted to make an `assert_jq_contains_line` where it would split the run command by line breaks and then assert that one of those lines exactly matches the match input. However, my jq and bash skills were not enough to figure it out in about 15-20 minutes, so I gave up for the time being. `assert_jq_not_contains` sounds easier, for sure.

Reply:
Updated now with `assert_jq_not_contains`